tests/console/rabbitmq.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    select_console 'user-console';
+    assert_script_sudo("zypper -n in rabbitmq-server");
+    assert_script_sudo("systemctl start rabbitmq-server");
+    assert_script_run("systemctl status rabbitmq-server");
```

Comment:
assert_script_run on a systemctl status will not show the journal logs of that service. Is that intentional?

If not, please use assert_script_sudo